src/main/java/com/jakewharton/rxrelay2/AppendOnlyLinkedArrayList.java

```diff
@@ -87,8 +87,12 @@ void forEachWhile(NonThrowingPredicate<? super T> consumer) {
         }
     }
 
+    /**
+     * Loops over all elements of the array until a null element is encountered.
+     * @param observer the observer of values
+     */
     @SuppressWarnings("unchecked")
-    boolean accept(Relay<? super T> observer) {
+    void forEachAccept(Relay<? super T> observer) {
```

Comment:
`accept` is the name in RxJava so I'd prefer to maintain the name to minimize the diff. Removing the return value is okay though.

Reply:
Done